packages/twenty-server/src/modules/favorite-folder/standard-objects/favorite-folder.workspace-entity.ts

```diff
@@ -0,0 +1,60 @@
+import { Relation } from 'src/engine/workspace-manager/workspace-sync-metadata/interfaces/relation.interface';
+
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+import { RelationMetadataType } from 'src/engine/metadata-modules/relation-metadata/relation-metadata.entity';
+import { BaseWorkspaceEntity } from 'src/engine/twenty-orm/base.workspace-entity';
+import { WorkspaceEntity } from 'src/engine/twenty-orm/decorators/workspace-entity.decorator';
+import { WorkspaceField } from 'src/engine/twenty-orm/decorators/workspace-field.decorator';
+import { WorkspaceRelation } from 'src/engine/twenty-orm/decorators/workspace-relation.decorator';
+import { FAVORITE_FOLDER_STANDARD_FIELD_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids';
+import { STANDARD_OBJECT_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-object-ids';
+import { FavoriteWorkspaceEntity } from 'src/modules/favorite/standard-objects/favorite.workspace-entity';
+
+@WorkspaceEntity({
+  standardId: STANDARD_OBJECT_IDS.favoriteFolder,
+  namePlural: 'favoriteFolders',
+  labelSingular: 'Favorite Folder',
+  labelPlural: 'Favorite Folders',
+  description: 'A Folder of favorites',
+  icon: 'IconFolder',
+})
+export class FavoriteFolderWorkspaceEntity extends BaseWorkspaceEntity {
+  @WorkspaceField({
+    standardId: FAVORITE_FOLDER_STANDARD_FIELD_IDS.position,
+    type: FieldMetadataType.NUMBER,
+    label: 'Position',
+    description: 'Favorite folder position',
+    icon: 'IconList',
+    defaultValue: 0,
+  })
+  position: number;
```

Comment:
Since we have `position` on 2 different objects (favoriteFolder and favorite) it will not possible to mix them in the frontend, which is fine (I checked with Thomas). So let's assume that folders are always displayed at the top, and favorites that are not in a folder are always display below. It won't be possible to do (folder 1 - favorite 1 - folder 2)